lib/code_sponsor_web.ex

```diff
@@ -60,6 +60,32 @@ defmodule CodeSponsorWeb do
     end
   end
 
+  def schema do
```

Comment:
Oh hm, I'm not sure if there's a best practice or standard around this. I'm in the habit of putting schemas in the same place as controllers/views like Rails does but this could be totally wrong, tbh.

Reply:
Generally, what I like to do is move `schema`s outside of the application namespace entirely with just `Schema.User` for instance but decided to make it slightly less disruptive for now, but I'm happy to update one way or the other if you'd like.

Reply:
If you aren't sure on best practice for this, I'll defer to the generators that Phoenix uses and say we put them in the code_sponsor folder. Would that be ok?

Reply:
Yeah I buy that, will update.